### DISCUSSION TITLE
[Script request] script.service.easyepg-lite

### DISCUSSION THREAD

**Discussion by CustomerJFK** [community-scripts/ProxmoxVE#915] [Request script] [answered]
### Application Name

script.service.easyepg-lite

### Website

https://github.com/sunsettrack4/script.service.easyepg-lite

### Description

Hello 
Can anyone be so kind to create a Proxmox LXC script for - script.service.easyepg-lite?

Thanks

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Add description for this and for what you doing. We discuss this after next Project Pipeline Workload

**Comment by CustomerJFK** [comment]
Hi 
It's a self-hosted EPG - grabber - tool. (based on Gracenote)
(Proof of Concept: XMLTV grabber tool using Gracenote's TMS API )
Thanks!

**Comment by btehan** [comment]
I'd really like to try this out sometime.

**Comment by tremor021** [comment, marked as answer]
Has no release tarballs on github, no updates in years... Nope @MickLesk
